azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -402,11 +403,10 @@ private void getToken(final String userToProxy) throws InterruptedException,
     }
   }
 
-  private void cancelHiveToken(final Token<? extends TokenIdentifier> t,
-      final String userToProxy) throws HadoopSecurityManagerException {
+  private void cancelHiveToken(final Token<? extends TokenIdentifier> t)
+      throws HadoopSecurityManagerException {
```

Comment:
I'd suggest you keep them separate in the future. It makes review harder. If you want a quicker review, keep the change as small and focused as possible.